native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe/deserializer/DeserializerFactory.cs

```diff
@@ -0,0 +1,49 @@
+using AWSGsrSerDe.common;
+using AWSGsrSerDe.deserializer.avro;
+
+namespace AWSGsrSerDe.deserializer
+{
+    /// <summary>
+    /// Factory to create a new instance of protocol specific de-serializer.
+    /// </summary>
+    public class DeserializerFactory
```

Comment:
Let's call this `DataFormatDeserializerFactory` and instance it creates `DataFormatDeserializer`

Reply:
Updated